### PR TITLE
Remove support for Py2

### PR DESCRIPTION
**Pull request by mcara**
CC: @jhunkeler @pllim

### CODE REVIEW

**Comment by pllim** [comment]
I'll defer to @jhunkeler on this one.

**Comment by mcara** [comment]
@pllim You can't do that! You started it all with https://github.com/spacetelescope/stsci.skypac/issues/39 😈

File: stsci.skypac/meta.yaml [hunk 1]
```diff
@@ -27,7 +28,7 @@ requirements:
     - numpy {{ numpy }}
     - python {{ python }}
     run:
-    - astropy >=1.1
+    - astropy >=3.1
```
Comment (pllim) [inline stsci.skypac/meta.yaml]: Strictly speaking, you can probably support `astropy >= 3` (which would include 3.0.x) unless you use `astropy` feature that only exists in 3.1 like APE14.

File: stsci.skypac/meta.yaml [hunk 2]
```diff
@@ -10,14 +10,15 @@ about:
 build:
     number: {{ number }}
     preserve_egg_dir: 'yes'
-
+    skip: true  # [py2k]
```
Comment (pllim) [inline stsci.skypac/meta.yaml]: Only @jhunkeler would know if the syntax is correct. Can there be a space after `#`? The original example he gave me did not have a space but maybe that does not matter.

File: stsci.skypac/meta.yaml (hunk 1 above)
Comment (mcara) [reply stsci.skypac/meta.yaml]: Yes, I need `3.1` (because of APE 14) - see `_naxis1` issues.

File: stsci.skypac/meta.yaml (hunk 2 above)
Comment (mcara) [reply stsci.skypac/meta.yaml]: Syntax should be fine (I think).

Comment (mcara) [reply stsci.skypac/meta.yaml]: @pllim 

> The original example he gave me did not have a space...

Why did you add it then???

Comment (pllim) [reply stsci.skypac/meta.yaml]: I did? Where?

Comment (mcara) [reply stsci.skypac/meta.yaml]: retire_py2k

Comment (mcara) [reply stsci.skypac/meta.yaml]: 🤣

Comment (pllim) [reply stsci.skypac/meta.yaml]: Haha... OK... Then LGTM!

**Comment by pllim** [comment]
But wait, you didn't bump the package version. That means this package version already is released into PY2 land. What is the first version that is PY3 only?

**Comment by mcara** [comment]
I know. The reason is that I also need to update documentation (which has build warnings), modify setup etc. I will **tag** the package and bump the version **after** all this work is done. However, `meta.yaml` should be changed now because build (from master) would fail. I am working on it. I will make another PR to `astroconda-contrib` to change version number once I tag `skypac`...

Well, while writing this, I realized that this maybe needs to be done in `dev`...

**Comment by pllim** [comment]
> maybe needs to be done in dev...

Yeah...

**Comment by mcara** [comment]
Version was updated to `1.0.0` and tagged.

**Comment by mcara** [comment]
If only I could merge this... BUT... essentially `drizzlepac` will need to be switched to Py3 only in `conda` recipes because it depends on `skypac`.

**Comment by pllim** [comment]
You could always also upgrade `drizzlepac` in this PR if they go together. I also cannot merge.

**Pull request merged**